t/cve-2017-3302.t

```diff
@@ -9,7 +15,7 @@ use vars qw($test_dsn $test_user $test_password $test_db);
 use lib 't', '.';
 require "lib.pl";
 eval {
-  $dbh = DBI->connect($test_dsn, $test_user, $test_password, {RaiseError => 1, mysql_server_prepare => 1});
```

Comment:
Hey!! What are you doing? Whole test needs to enable by default prepared statement as it tests CVE-2017-3302. Please revert whole changes in this file. It should test on every version, not just some affected!